DVR/Cassette.swift

```diff
@@ -18,16 +18,16 @@ struct Cassette {
 
     // MARK: - Functions
 
-    func interactionForRequest(request: NSURLRequest, ignoreBaseURL: Bool) -> Interaction? {
+    func interactionForRequest(_ request: URLRequest, ignoreBaseURL: Bool) -> Interaction? {
         for interaction in interactions {
             let interactionRequest = interaction.request
 
-            guard let interactionURL = interactionRequest.URL,
-                requestURL = request.URL else {
+            guard let interactionURL = interactionRequest.url,
+                let requestURL = request.url else {
                 return nil
             }
             // Note: We don't check headers right now
-            if interactionRequest.HTTPMethod == request.HTTPMethod && interactionURL.isEqualWithURL(requestURL, ignoreBaseURL: ignoreBaseURL) && interactionRequest.hasHTTPBodyEqualToThatOfRequest(request)  {
+            if interactionRequest.httpMethod == request.httpMethod && interactionURL.isEqual(toUrl: requestURL, ignoreBaseURL: ignoreBaseURL) && interactionRequest.hasHTTPBodyEqualToThatOfRequest(request)  {
```

Comment:
Parameter `toUrl` i thing should be `toURL`.
Note: If you just pulled the PR from DVR and added on our repo i think we shouldn't modify this.